### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by timvdstap**
Minor updates to the General Information page

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

You may delete these instructions from your comment.

\- Marine Biological Data Mobilization Workshop Maintainers
</details>

### CODE REVIEW

**Comment by MathewBiddle** [comment]
These changes look good. We just need to discuss the dates in here. I prefer the website says we end on the 23 with optional 24 discussed during the workshop.

File: index.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ country: ""           # lowercase two-letter ISO country code such as "fr" (see
 language: "en"        # lowercase two-letter ISO language code such as "fr" (see https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) for the workshop
 latitude: 0          # decimal latitude of workshop venue (use https://www.latlong.net/)
 longitude: 0         # decimal longitude of the workshop venue (use https://www.latlong.net)
-humandate: "April 22-23, 2024"    # human-readable dates for the workshop (e.g., "Feb 17-18, 2020")
+humandate: "April 22-24, 2024"    # human-readable dates for the workshop (e.g., "Feb 17-18, 2020")
```
[the remark this answers is not in the document]
Comment (MathewBiddle) [reply index.md:12]: @timvdstap do you mind reverting this back?

File: index.md [hunk 2]
```diff
@@ -9,10 +9,10 @@ country: ""           # lowercase two-letter ISO country code such as "fr" (see
...
 humantime: "13:00 - 17:00 EDT"    # human-readable times for the workshop e.g., "9:00 am - 4:30 pm CEST (7:00 am - 2:30 pm UTC)"
 startdate: 2023-04-22      # machine-readable start date for the workshop in YYYY-MM-DD format like 2015-01-01
-enddate: 2023-04-23        # machine-readable end date for the workshop in YYYY-MM-DD format like 2015-01-02
+enddate: 2023-04-24        # machine-readable end date for the workshop in YYYY-MM-DD format like 2015-01-02
```
Comment (MathewBiddle) [inline index.md:15]: this too

File: index.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ INTRODUCTION
 Edit the general explanatory paragraph below if you want to change
 the pitch.
 {% endcomment %}
-This workshop is intended to be a small hands-on, interactive virtual workshop focused on mobilizing marine biological observation datasets to the <a href="https://obis.org/">Ocean Biodiversity Information System (OBIS)</a> by helping data providers standardize their data using Darwin Core including species observations from any type of sampling methodologies (e.g. visual surveys, net tows, microscopy, fish trawls, imaging, ‘omics, acoustics, telemetry).
+This workshop is intended to be a small hands-on, interactive virtual workshop focused on mobilizing marine biodiversity observation datasets to the <a href="https://obis.org/">Ocean Biodiversity Information System (OBIS)</a> by helping data providers standardize their data using Darwin Core including species observations from any type of sampling methodologies (e.g. visual surveys, net tows, microscopy, fish trawls, imaging, ‘omics, acoustics, telemetry).
```
Comment (MathewBiddle) [inline index.md:94]: this too

File: index.md [hunk 4]
```diff
@@ -389,7 +391,7 @@ of code below the Schedule `<h2>` header below with
 <h2 id="schedule">Schedule</h2>
 __Times are in EDT.__
 
-Overview recordings of each lesson can be found in [this playlist](https://youtube.com/playlist?list=PLlgUwSvpCFS7zytaWbZ6f4Szm3PnpFj_J).
+The videos created for the Marine Biodiversity Data Mobilization Workshop can be found in [this playlist](https://youtube.com/playlist?list=PLlgUwSvpCFS7zytaWbZ6f4Szm3PnpFj_J).
```
Comment (MathewBiddle) [inline index.md:394]: Adjust to Biological

**Review by MathewBiddle — CHANGES_REQUESTED** [review]
@timvdstap do you mind reverting back a few of these changes. I really like your emphasis and text additions here.

**Comment by jdpye** [comment]
my read is that this is still waiting on @timvdstap to twiddle the dates back ? if not, it's mergeable.

File: index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (timvdstap) [reply index.md]: Sorry not sure how this slipped under the radara - resolved now back to April 23.

**Review by MathewBiddle — APPROVED** [review]

**Pull request merged**